plugins/source/aws/resources/services/organizations/organizations_fetch.go

```diff
@@ -0,0 +1,23 @@
+package organizations
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchOrganizationsOrganizations(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	o, err := c.Services().Organizations.DescribeOrganization(ctx, &organizations.DescribeOrganizationInput{})
+	if err != nil {
+		return err
+	}
+	if o == nil || o.Organization == nil {
+		return nil
+	}
```

Comment:
Is this what happens when an AWS account is not in an Org?

Reply:
Apparently not, removed. You get a `AWSOrganizationsNotInUseException`, same with ListAccounts.